R/run_trisk.R

```diff
@@ -80,6 +80,12 @@ run_trisk <- function(input_path,
     risk_type = "trisk"
   )
 
+  infer_scenario_type(
```

Comment:
1. I would suggest to already assign what gets returned here to a new object called scenario_type
2. also needs to be in run_lrisk